public/app/features/scenes/components/ScenePanelRepeater.tsx

```diff
@@ -21,7 +22,7 @@ export class ScenePanelRepeater extends SceneObjectBase<RepeatOptions> {
     super.activate();
 
     this._subs.add(
-      this.getData().subscribeToState({
+      sceneGraph.getDataFor(this).subscribeToState({
```

Comment:
alternative name getDataFrom,  getTimeRangeFrom, interpolateFrom etc?  or skip the For/From suffix

Reply:
`getData` and others without suffix are just fine for me, TS will tell the rest of the story via types.